### ISSUE TITLE
Crashes happens from CustomFileValidator due to NullPointerException

### ISSUE THREAD

**Issue by gouri-panda** [kiwix/kiwix-android#3663]
**Describe the bug**
Reported by goodle from Wikimed
**Expected behavior**
Shouldn't crash
**Logs**
```
Exception java.lang.NullPointerException:
  at org.kiwix.kiwixmobile.custom.main.CustomFileValidator.zimFiles (CustomFileValidator.kt:46)
  at org.kiwix.kiwixmobile.custom.main.CustomReaderFragment.openObbOrZim (CustomReaderFragment.kt:24)
  at org.kiwix.kiwixmobile.custom.main.CustomReaderFragment.onViewCreated (CustomReaderFragment.kt:196)
  at androidx.fragment.app.FragmentStateManager.createView (FragmentStateManager.java:254)
  at androidx.fragment.app.FragmentStateManager.moveToExpectedState (FragmentStateManager.java:146)
  at androidx.fragment.app.FragmentManager.moveToState (FragmentManager.java:58)
  at androidx.fragment.app.FragmentManager.dispatchStateChange (FragmentManager.java:34)
  at androidx.fragment.app.FragmentStateManager.createView (FragmentStateManager.java:259)
  at androidx.fragment.app.FragmentStateManager.moveToExpectedState (FragmentStateManager.java:146)
  at androidx.fragment.app.FragmentManager.moveToState (FragmentManager.java:58)
  at androidx.fragment.app.FragmentManager.dispatchStateChange (FragmentManager.java:34)
  at androidx.fragment.app.FragmentActivity.onStart (FragmentActivity.java:32)
  at androidx.appcompat.app.AppCompatActivity.onStart (AppCompatActivity.java:1)
  at org.kiwix.kiwixmobile.core.main.CoreMainActivity.onStart (CoreMainActivity.kt:1)
  at org.kiwix.kiwixmobile.custom.main.CustomMainActivity.onStart (CustomMainActivity.kt:1)
  at android.app.Instrumentation.callActivityOnStart (Instrumentation.java:1529)
  at android.app.Activity.performStart (Activity.java:8447)
  at android.app.ActivityThread.handleStartActivity (ActivityThread.java:3892)
  at android.app.servertransaction.TransactionExecutor.performLifecycleSequence (TransactionExecutor.java:221)
  at android.app.servertransaction.TransactionExecutor.cycleToPath (TransactionExecutor.java:201)
  at android.app.servertransaction.TransactionExecutor.executeLifecycleState (TransactionExecutor.java:173)
  at android.app.servertransaction.TransactionExecutor.execute (TransactionExecutor.java:97)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:2465)
  at android.os.Handler.dispatchMessage (Handler.java:106)
  at android.os.Looper.loopOnce (Looper.java:240)
  at android.os.Looper.loop (Looper.java:351)
  at android.app.ActivityThread.main (ActivityThread.java:8370)
  at java.lang.reflect.Method.invoke
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:568)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1013)
```
**Screenshots**

**Environment**
- Version of Kiwix Android : 6231840
- Device : Realme
- OS version : Android 13

**Comment by MohitMaliFtechiz** [comment]
@gouri-panda I am trying to reproduce this error, but I am not able to reproduce this.

The version of Kiwix you have reported is very old, the latest version of Kiwix is `6233630`. In between these versions, we have improved our file loading functionality and the `CustomFileValidator.zimFiles` method too.

See https://github.com/kiwix/kiwix-android/commit/035d56d15282c912587faaaf7d4247e9b84e3ee0 for more details, in this we have improved our `zimFiles` method to avoid this type of `NullPointerException` error.

Is this error happening with the latest version of Kiwix?
